org.eclipse.jdt.debug.tests/build.properties

```diff
@@ -26,7 +26,8 @@ bin.includes = plugin.xml,\
                java8/,\
                java9/,\
                java16_/,\
-               java22/
+               java22_/,\
```

Comment:
This line is the problem

Reply:
Thanks, which log did you find this in? Looks like the one I reported is a red herring then.

Reply:
https://ci.eclipse.org/jdt/job/eclipse.jdt.debug-github/job/PR-482/16/console